utils/spdx/build.gradle.kts

```diff
@@ -286,30 +286,16 @@ fun generateEnumClass(
 }
 
 val fixupLicenseTextResources by tasks.registering {
-    val licenseUrlMap: Map<File, URL>? by project.extra
-    val licenseExceptionUrlMap: Map<File, URL>? by project.extra
-
     doLast {
-        (licenseUrlMap.orEmpty() + licenseExceptionUrlMap.orEmpty()).forEach { (targetFile, url) ->
-            val filename = url.path.substringAfterLast('/')
-
-            val downloadedFile = targetFile.resolveSibling(filename)
-
-            // Rename files that are named differently on the server.
-            if (downloadedFile != targetFile) {
-                if (targetFile.exists() && !targetFile.delete()) {
-                    throw GradleException("Failed to delete existing file '$targetFile'.")
-                }
-
-                if (!downloadedFile.renameTo(targetFile)) {
-                    throw GradleException("Failed to rename file '$downloadedFile' to '$targetFile'.")
-                }
+        val resourcePaths = listOf("licenses", "exceptions").map { file("src/main/resources/$it") }
+
+        resourcePaths.forEach { path ->
+            path.listFiles().forEach { file ->
+                // Trim trailing whitespace and blank lines.
+                val lines = file.readLines().map { it.trimEnd() }.asReversed().dropWhile { it.isEmpty() }
```

Comment:
I needed to look a while at lines 294 - 296 to understand what it's doing, and I still don't understand why `asReversed` is used (twice). I believe factoring out that code into a function with a speaking name would solve that problem and make the code comment (at least in this place) redundant.

Reply:
Please note that this code has been there for quite some time and was just moved multiple times recently. I'll still have a look how to improve it, but in a new commit.

Reply:
Oh sorry, I missed that. Please disregard my comment then.